qemu/tests/openflow_acl_test.py

```diff
@@ -95,20 +95,18 @@ def access_service(access_sys, access_targets, disabled, host_ip,
                         check_string = access_params.get("check_from_output")
                         if check_string and check_string in out:
                             stat = 1
-                    except (aexpect.ShellCmdError, error.CmdError,
-                            aexpect.ShellTimeoutError), err:
-                        if isinstance(err, error.CmdError):
-                            out = err.result_obj.stderr
-                            stat = err.result_obj.exit_status
-                        else:
-                            out = err.output
-                            if isinstance(err, aexpect.ShellTimeoutError):
-                                stat = 1
-                                session.close()
-                                session = vm.wait_for_login(timeout=timeout)
-                                run_func = session.cmd
-                            else:
-                                stat = err.status
+                    except aexpect.ShellCmdError as err:
+                        out = err.output
+                        stat = err.status
+                    except aexpect.ShellTimeoutError as err:
+                        stat = 1
```

Comment:
it seems `aexpect.ShellTimeoutError` needs a `out = err.output` as well

Reply:
Updated, thanks.